test/fuzz/param.ml

```diff
@@ -14,3 +14,7 @@ let initial_fuel = 100
 let allow_partial_timeout = true
 
 let max_time_execution = 0.01 (* seconds *)
+
+let save_modules = true (* Set to false to disable saving modules *)
```

Comment:
I would set this to false by default because this is quite likely to make the fuzzing much more slower